internal/cmd/load-balancer/target-pool/describe/describe.go

```diff
@@ -0,0 +1,189 @@
+		Long:  "Shows details of a target pool in a Load Balancer.",
+		Args:  args.SingleArg(targetPoolNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a target pool with name "pool" in load balancer with name "my-load-balancer"`,
+				"$ stackit load-balancer target-pool describe pool --lb-name my-load-balancer"),
+			examples.NewExample(
+				`Get details of a target pool with name "pool" in load balancer with name "my-load-balancer in JSON output"`,
+				"$ stackit load-balancer target-pool describe pool --lb-name my-load-balancer --output-format json"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read load balancer: %w", err)
+			}
+
+			return outputResult(p, model, resp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().String(lbNameFlag, "", "Name of the load balancer")
+
+	err := flags.MarkFlagsRequired(cmd, lbNameFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	targetPoolName := inputArgs[0]
+
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		TargetPoolName:  targetPoolName,
+		LBName:          cmd.Flag(lbNameFlag).Value.String(),
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *loadbalancer.APIClient) loadbalancer.ApiGetLoadBalancerRequest {
+	req := apiClient.GetLoadBalancer(ctx, model.ProjectId, model.LBName)
+	return req
+}
+
+func outputResult(p *print.Printer, model *inputModel, loadBalancer *loadbalancer.LoadBalancer) error {
+	switch model.OutputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(loadBalancer, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal load balancer: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		return outputResultAsTable(p, model.TargetPoolName, loadBalancer)
+	}
+}
+
+func outputResultAsTable(p *print.Printer, targetPoolName string, loadBalancer *loadbalancer.LoadBalancer) error {
+	targetPool := utils.FindLoadBalancerTargetPoolByName(loadBalancer.TargetPools, targetPoolName)
```

Comment:
Because currently this `targetPool` could be `nil` here, and in this case we shouldn't output anything and should return an error